plugins/inventory/proxmox.py

```diff
@@ -226,6 +226,10 @@ def _get_vm_config(self, node, vmid, vmtype, name):
                 if config == 'rootfs' or config.startswith(('virtio', 'sata', 'ide', 'scsi')):
                     value = ('disk_image=' + value)
 
+                # Parse Proxmox tag string into list
+                if config == 'tags':
+                    value = [tag.strip() for tag in value.split(",")]
```

Comment:
This is a breaking change, as it modifies the behavior of the current `<prefix>tags` fact.

It would be better to add this as a new fact.

Reply:
You are correct. I misunderstood the current behavior.
I have revised the changes as a bugfix to avoid parsing tags strings as dicts since tags strings do not permit "=" sign characters.
This will allow tags strings containing commas to be passed normally as facts.
Any additional parsing should be done via jinja and the constructed plugin as proxmox does not provide a data structure for the tags field.

Reply:
Additional field for parsed tags added after further discussion